app/boards/arm/dz60rgb/dz60rgb_rev1.yaml

```diff
@@ -17,3 +17,6 @@ supported:
   - nvs
   - can
   - kscan
+manufacturer: dztech
+standalone: true
```

Comment:
The meaning of `standalone` didn't jump out at me which means it probably won't for others either.  It seems to be `true` when `compatible` is blank?  So, `shieldless` I guess?  But the type of `bdn9_rev2` is `mcu` not `keyboard`, which confuses my limited understanding.

Reply:
Yeah if there's no compatible you *could* call it standalone with the current set. Although there's a chance support is added for a board that has no compatible shield interconnect but isn't a standalone board. That would be a board used for handwires.

Reply:
I think I understand where you're coming from.  So "standalone" is mostly indicating that the board is a functioning keyboard/device out-of-the-box.

We need to ensure boards which aren't standalone and don't fit into named scheme are handled by the hardware page, i.e. miscellaneous/other for lack of a better term.